lib/chef/provider/package/chocolatey.rb

```diff
@@ -130,6 +130,19 @@ def remove_package(names, versions)
         # install from, but like the rubygem provider's sources which are more like repos.
         def check_resource_semantics!; end
 
+        def get_choco_version
+          @get_choco_version ||= begin
```

Comment:
should just be able to `||= powershell_exec!("choco --version").result` here @johnmccrae